README.md

```diff
@@ -50,38 +50,55 @@ If your app is using some of the features of Swift 6, we recommend setting the *
 ```swift
 import Confidence
 
-let confidence = Confidence.Builder(clientSecret: "mysecret", loggerLevel: .NONE).build()
+let confidence = Confidence
+   .Builder(clientSecret: "mysecret", loggerLevel: .NONE)
+   .withContext(context: ["user_id": ConfidenceValue(string: "user_1")])
+   .build()
 await confidence.fetchAndActivate()
 ```
 
 - The `clientSecret` for your application can be generated in the Confidence portal.
 - The `loggerLevel` sets the verbosity level for logging to console. This can be useful while testing your integration with the Confidence SDK.
+- `withContext()` sets the initial context. The context is a key-value map used for sampling and for targeting, so it determines how flags are evaluated by the Confidence backend.
 
 _Note: the Confidence SDK has been intended to work as a single instance in your Application.
 Creating multiple instances in the same runtime could lead to unexpected behaviours._
 
 ### Initialization strategy
 
-`confidence.activateAndFetch()` is an async function that fetches the flags from the Confidence backend,
-stores the result on disk, and make the same data ready for the Application to be consumed.
+After creating the confidence instance, you can choose between different strategies to initialize the SDK:
+- `await confidence.fetchAndActivate()`: async function that fetches the flags from the Confidence backend according to the current context,
+stores the result in storage, and make the same data ready for the Application to be consumed.
 
-The alternative option is to call `confidence.activate()`: this loads previously fetched flags data
+- `confidence.activate()`: this loads previously fetched flags data
 from storage and makes that available for the Application to consume right away.
-To avoid waiting on backend calls when the Application starts, the suggested approach is to call
-`confidence.activate()` and then trigger a background refresh via `confidence.asyncFetch()` for future sessions.
 
-### Setting the context
-The context is a key-value map used for sampling and for targeting, when flag are evaluated by the Confidence backend.
-It is also appended to the tracked events, making it a great way to create dimensions for metrics in Confidence.
+If you wish to avoid waiting on backend calls when the Application starts, the suggested approach is to call
+`confidence.activate()` and then call `confidence.asyncFetch()` to update the flag values in storage to be used on a future `activate()`. However, note that `activate()` ignores the current context: even if the current context has changed since the last fetch, flag values from the last fetch will be exposed to the Application.
```

Comment:
I realised when reading this (combined with thinking about `app version` as a context value) that "multi context storage" would probably make us return default values after an app version update. 

Future thinking work: Should we have support for parts of context that do not affect context storage location?

Reply:
I think having different parts of contexts behaving differently according to their importance/use-case could enable better integrations, definitely something to look into in the future